equinix/resource_fabric_cloud_router.go

```diff
@@ -132,9 +150,20 @@ func fabricCloudRouterResourceSchema() map[string]*schema.Schema {
 				Schema: fabricCloudRouterProjectSch(),
 			},
 		},
+		"marketplace_subscription": {
+			Type:        schema.TypeSet,
+			Optional:    true,
+			Computed:    true,
+			Description: "Equinix Fabric Entity for Marketplace Subscription",
+			MaxItems:    1,
+			Elem: &schema.Resource{
+				Schema: fabricMarketplaceSubscriptionSch(),
+			},
+		},
 		"account": {
 			Type:        schema.TypeSet,
-			Required:    true,
+			Optional:    true,
```

Comment:
Same question about account. I believe this is correct but double checking.

Reply:
account number is required in the account filed and I have marked it as required in schema definition.